.gitignore

```diff
@@ -1,56 +1,35 @@
-logs/*.tar.*
 etc/mine/
```

Comment:
This folder is referenced here: https://open.qa/docs/#_customize_configuration_directory
So I left it in.